ouroboros-consensus/src/Ouroboros/Consensus/Ledger/Byron/Forge.hs

```diff
@@ -191,7 +190,8 @@ forgeRegularBlock cfg curSlot curNo prevHash txs isLeader = do
 
     prevHeaderHash :: CC.Block.HeaderHash
     prevHeaderHash = case prevHash of
-      GenesisHash             -> CC.Block.genesisHeaderHash pbftGenesisHash
+      GenesisHash             -> error
+        "forging a regular block instead of an EBB on top of an empty chain"
```

Comment:
Slightly confusing error message. Maybe "the first block on the Byron ledger _must_ be an EBB".